kolibri/core/assets/src/utils/licenseTranslations.js

```diff
@@ -236,6 +236,13 @@ export function licenseLongName(leUtilsLicenseName) {
 
 // Translated license descriptions, aimed at creators of the content
 export function licenseDescriptionForCreator(leUtilsLicenseName, leUtilsLicenseDescription) {
```

Comment:
Where in Kolibri do you view this "creator" version? I'm assuming the "consumer" version is what's on the Content Page.

Reply:
actually, this doesn't look like function is used anywhere in the code

Reply:
Yeah, I grepped for it and found no references. I thought it best to just make sure the implementation was parallel. I assume this was included in a forward looking way to offline content creation license application.